cibuildwheel/docker_container.py

```diff
@@ -31,19 +31,26 @@ class DockerContainer:
     bash_stdout: IO[bytes]
 
     def __init__(
-        self, *, docker_image: str, simulate_32_bit: bool = False, cwd: Optional[PathOrStr] = None
+        self,
+        *,
+        docker_image: str,
+        simulate_32_bit: bool = False,
+        network_host: bool = False,
+        cwd: Optional[PathOrStr] = None,
     ):
         if not docker_image:
             raise ValueError("Must have a non-empty docker image to run.")
 
         self.docker_image = docker_image
         self.simulate_32_bit = simulate_32_bit
+        self.network_host = network_host
         self.cwd = cwd
         self.name: Optional[str] = None
 
     def __enter__(self) -> "DockerContainer":
         self.name = f"cibuildwheel-{uuid.uuid4()}"
         cwd_args = ["-w", str(self.cwd)] if self.cwd else []
+        network_args = ["--network=host"] if self.network_host else []
```

Comment:
I'd make this check for Travis CI + ppc64le, and only apply it there. In fact, you probably could/should check for native ppc64le rather than checking the target (cross compiles are unaffected)

Reply:
Happy to do that. Can you guide me on this please?

- As travis detection, just use `if "TRAVIS" in os.environ`?
- How to check for native ppc64le in cibuildwheel?

Thanks! :)

P.S.: If it's faster to push than answer then feel free to also update as you see fit - both good for me :)

Reply:
`cibuildwheel.util.detect_ci_provider() == cibuildwheel.util.CIProvider.travis_ci`

And `platform.machine()` for the platform?

Reply:
Alright, thanks for the guidance! I moved the work-around directly at the current location and removed the interface control for it.